.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,40 @@
+---
+name: 🐛 Bug Fix
+about: Did you fix something that did not work as expected?
+---
+
+<!---
+Thanks for filing a pull request 😄 ! Before you submit, please read the following:
+
+Search open/closed issues before submitting since someone might have pushed the same thing before!
+-->
+
+# 🐛 bug fix
+
+<!--- Provide the details of the pull request here -->
+
+## 🤔 What was not working?
+
+<!--- Tell us what should happen -->
```

Comment:
This should probably be `tell us what goes wrong` or split it up into 2 steps like the issue template